src/common/mod.rs

```diff
@@ -568,7 +572,7 @@ mod tests {
 
     fn unset_envvar(varname: &String) {
         let _ = unsafe {
-            libc::unsetenv(varname.as_ptr() as *const i8);
+            libc::unsetenv(varname.as_ptr() as *const c_char);
```

Comment:
why is this change needed? 

```
pub type c_char = i8;
```
[https://doc.rust-lang.org/std/os/raw/type.c_char.html](https://doc.rust-lang.org/std/os/raw/type.c_char.html)

Reply:
If you read the text below:
This type will always be either i8 or u8, as the type is defined as being one byte long.

Reply:
https://github.com/rust-lang/rust/issues/79089

Reply:
I got it!